code/modules/ninja/ninja_event.dm

```diff
@@ -90,4 +90,4 @@ Contents:
 	A.real_name = "[pick(GLOB.ninja_titles)] [pick(GLOB.ninja_names)]"
 	A.copy_to(new_ninja)
 	new_ninja.dna.update_dna_identity()
-	return new_ninja
+	return new_ninja
```

Comment:
This needs a newline.

Reply:
This is just do to editing the file once and then going back on it, nothing is changed